main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+    "encoding/csv"
+    "fmt"
+    "net/http"
+    "os"
+    "sync/atomic"
+)
+
+var counter uint64
+
+func main() {
+    http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+        fmt.Println("Received request:")
+        writer := csv.NewWriter(os.Stdout)
+        defer writer.Flush()
+        writer.Write([]string{"Header", "Value"})
+        for k, v := range r.Header {
+            writer.Write([]string{k, v[0]})
```

Comment:
When there are multiple values corresponding to the same key, output all values along with their keys. For example:

```
Header,Value
k1,v2
k1,v3
k2,v4
k2,v5
```